components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -194,7 +195,18 @@ - (void)commonMDCTextInputCommonFundamentInit {
 
 - (void)setupClearButton {
   if (!_clearButton) {
-    _clearButton = [[UIButton alloc] initWithFrame:CGRectZero];
+    // The following MDCButton configuration creates an MDCButton that mimics a UIButton with a
+    // "clear" icon and adds a ripple effect.
+    MDCButton *clearButton = [[MDCButton alloc] init];
+    clearButton.backgroundColor = UIColor.clearColor;
+    // This ink color was taken from the MDCButton+MaterialTheming behavior, with UIColor.blackColor
+    // taken from the onSurfaceColor value of the MDCColorSchemeDefaultsMaterial201907 color scheme.
+    clearButton.inkColor = [UIColor.blackColor colorWithAlphaComponent:(CGFloat)0.12];
```

Comment:
It looks like there should be a public API for clients to control the ripple color. If their Ripple is a non-gray color throughout their app, this Ripple won't be able to match the other components.  

Would you please add a public API for ripple color in a follow-up PR?

Reply:
Great idea, will do

Reply:
#8794